tests/test_loader.py

```diff
@@ -14,7 +14,7 @@
 from preggy import expect
 from thumbor.config import Config
 from tornado.testing import gen_test
-
+from mock import patch
```

Comment:
You need to replace this import with `from unittest.mock import patch`

Reply:
Done! thanks for the pointer